ruma-client-api/src/r0/media/create_content.rs

```diff
@@ -30,6 +30,14 @@ ruma_api! {
     response: {
         /// The MXC URI for the uploaded content.
         pub content_uri: String,
+
+        /// The [BlurHash](https://blurha.sh) for the uploaded content.
+        ///
+        /// This uses the unstable prefix in MSC2448.
+        #[cfg(feature = "unstable-pre-spec")]
+        #[serde(rename = "xyz.amorgan.blurhash")]
+        #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Please merge these two attributes:

```suggestion
        #[serde(rename = "xyz.amorgan.blurhash", skip_serializing_if = "Option::is_none")]
```